src/main/java/ro/societateahermes/backendservice/service/serviceImplementation/FormService.java

```diff
@@ -0,0 +1,38 @@
+package ro.societateahermes.backendservice.service.serviceImplementation;
+
+import org.springframework.stereotype.Service;
+import ro.societateahermes.backendservice.entities.form.Form;
+import ro.societateahermes.backendservice.entities.form.Option;
+import ro.societateahermes.backendservice.entities.form.Question;
+import ro.societateahermes.backendservice.repository.FormRepository;
+
+import java.util.List;
+
+@Service
+public class FormService {
+    private final FormRepository formRepository;
+
+    public FormService(FormRepository formRepository) {
```

Comment:
Foloseste @Autowired cand declari constructorul sau field-ul. Asa spring o sa iti injecteze automat obiectul de care vei avea nevoie

Reply:
Incepand cu Spring Boot 1.4 constructorii au automat @Autowired daca clasa are un singur constructor:

https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-spring-beans-and-dependency-injection.html

Sa adaug totusi anotarea oricum pentru claritate?

Reply:
Daca e automat nu. Lasa cum e acum